tico/serialize/operators/op_conv2d.py

```diff
@@ -120,7 +120,7 @@ def define_node(self, node: torch.fx.Node) -> circle.Operator.OperatorT:
         conv_input: torch.fx.node.Node | circle.Tensor.TensorT = input_
         weight_shape = list(extract_shape(weight))
 
-        if is_same_padding(padding, input_shape, output_shape):
+        if is_same_padding(padding, input_shape, output_shape) and stride == [1, 1]:
```

Comment:
How about changing the order?
```py
if is_valid_padding(...):
  ...
elif is_same_padding(...):
```
If padding is specified as "valid", it should be "valid".
But if we check `is_same_padding` first, "valid" may happen to be guessed as "same".